parse.y

```diff
@@ -907,6 +914,11 @@ class_item /* IEEE1800-2005: A.1.8 */
   | method_qualifier_opt function_declaration
       { /* The function_declaration rule puts this into the class */ }
 
+  | pure_virtual_method_decl
+      { yywarn(@1, "OOP Pure Virtual methods are supported as parse-only. "
+                   "No enforcement is done for a concrete class to define these further");
+      }
+
```

Comment:
We really need a "Sorry" message here, and not a warning. Since the feature is supported, it should generate an error, and a "Sorry" message (see examples elsewhere in the code) is the convention for expressing that.